Update handbook.md

csalzman opened the pull request:
Updating go!pass section

csalzman committed:
Update handbook.md

Updating go!pass section

twbrandt reviewed (changes requested):
It's now the Ann Arbor Area Transportation Authority (AAATA).

cdzombak reviewed (dismissed):
"Transprotation" is misspelled

csalzman committed:
Update handbook.md

Fixing typo and adding the appropriate number of 'A' words to The Ride's official name.
